libs/stylelint-config/package.json

```diff
@@ -23,7 +23,7 @@
     "stylelint-group-selectors": "^1.0.10"
   },
   "peerDependencies": {
-    "stylelint": "^16.10.0"
+    "stylelint": "^16.12.0"
```

Comment:
Should we consider adding the selector-pseudo-class-allowed-list rule to our styling configuration? Since this rule has been extended in the newest version of our stylint.

Reply:
...And even if we don't want it, we still miss it in our config, so - yes.